### PR TITLE
247 org page template

### PR DESCRIPTION
**Pull request by rosexw**
This is for issue: "Create page template for organization page #247"
http://localhost:8080/#/organization/123/org-name
A new page is created for this and it only shows a header for us to add the details when we receive them later.

### CODE REVIEW

File: app/config/routes.js [hunk 1]
```diff
@@ -69,6 +70,8 @@ const routes = (
           <Route path="/get-started/social" component={Social} />
           <Route path="/get-started/email-settings" component={EmailSettings} />
           <Route path="/speaker/:id/:fullName?" component={Speaker} />
+          <Route path="/blah" component={Speaker} />
```
Comment (s-kennedy) [inline app/config/routes.js]: do we really need this lol

File: app/pages/Onboarding/Profile/Profile.js [hunk 2]
```diff
@@ -131,6 +131,36 @@ const Profile = props => {
           </RadioGroup>
         </FormField>
 
+        <FormField fullWidth className={css.formControl}>
```
Comment (s-kennedy) [inline app/pages/Onboarding/Profile/Profile.js]: This is from a different PR, right?

File: app/pages/Home/components/Filters.js [hunk 3]
```diff
@@ -149,16 +149,28 @@ class Filters extends Component {
 
             switch (identity.label) {
               case 'All speakers':
-                selected = !searchParams['poc'] && !searchParams['woman'];
+                selected = !searchParams['poc'] && !searchParams['woman'] && !searchParams['lgbtqa'];
```
Comment (s-kennedy) [inline app/pages/Home/components/Filters.js]: This is from a different PR right?

**Review by s-kennedy — CHANGES_REQUESTED** [review]

**Comment by rosexw** [comment]
@s-kennedy, fixed!

**Comment by s-kennedy** [comment]
@rosexw Thank you!!

**Pull request merged**